database/events/json_forms.py

```diff
@@ -38,6 +40,10 @@ def create_wtf_field(name, schema, required=True):
     if schema['type'] == 'boolean':
         return wtforms.BooleanField(**kwargs)
 
+    if schema['type'] == 'latex':
```

Comment:
shouldn't this by `{'type': 'string', 'format': 'latex'}? Jsonschema has no type `latex`

Reply:
See: https://json-schema.org/understanding-json-schema/reference/string.html

There are several predefined formats, e.g. email or date.

Reply:
See e.g. https://github.com/pep-dortmund/member-database/pull/38

Reply:
Ok thanks yes that makes sense.

Reply:
Just made the fix

Reply:
Cool! Sorry for being slow. Can't spent as much time as I would like to right now 😁